### PR TITLE
Update Microsoft Word and Excel

### PR DESCRIPTION
**Pull request by ZhmieXD**
## Description

I actually didn't plan to add or update more icons anytime soon but I have a question about my fork so I might as well ask it while doing another icon update I've been thinking of doing every time I look back to this project lately.

The question is, I saw pull requests from dependabot in my fork recently. What should I do? Does it have something to do with the fact that the app force closed when I ran it on my phone after ADB install recently? The icons still apply and work fine though, but when I launched the app from the app drawer, it crashes.

Otherwise, thanks!

## Type of change

:x: Bug fix (non-breaking change which fixes an issue)
:white_check_mark: Icon addition (non-breaking change that adds/modifies Lawnicons's icons)
:x: General change (non-breaking change that doesn't fit the above categories like copyediting)

### Icons updated
* Microsoft Excel (`com.microsoft.office.excel`)
* Microsoft Word (`com.microsoft.office.word`)

### CODE REVIEW

**Comment by SuperDragonXD** [comment]
> The question is, I saw pull requests from dependabot in my fork recently. What should I do?

You can simply merge them if you want, or close them. Dependabot simply updates... well, dependencies.

> Does it have something to do with the fact that the app force closed when I ran it on my phone after ADB install recently? The icons still apply and work fine though, but when I launched the app from the app drawer, it crashes.

If it's a `ResourcesNotFoundException`, see #834; rebase (sync changes) to fix that in your fork. Otherwise not sure unless you open an issue and explain it.

**Comment by ZhmieXD** [comment]
> > The question is, I saw pull requests from dependabot in my fork recently. What should I do?
> 
> You can simply merge them if you want, or close them. Dependabot simply updates... well, dependencies.
> 
> > Does it have something to do with the fact that the app force closed when I ran it on my phone after ADB install recently? The icons still apply and work fine though, but when I launched the app from the app drawer, it crashes.
> 
> If it's a `ResourcesNotFoundException`, see #834; rebase (sync changes) to fix that in your fork. Otherwise not sure unless you open an issue and explain it.

I tried to look it up but I accidentally closed the crash dialog instead, will try again later

**Review by Chefski — COMMENTED** [review]
Hey! Icons look good. Can't merge though as its failing 2 checks

File: build.gradle.kts [hunk 1]
```diff
@@ -1,7 +1,7 @@
 plugins {
     id("com.android.application") version "7.3.1" apply false
     id("com.android.library") version "7.3.1" apply false
-    id("org.jetbrains.kotlin.android") version "1.7.20" apply false
+    id("org.jetbrains.kotlin.android") version "1.7.21" apply false
```
Comment (SuperDragonXD) [inline build.gradle.kts:4]: Please unbump the version number. The Dependabot PR in your repo (`#3`) shows that the checks fail (which also applies in this PR) so you shouldn't merge them if they do.

**Review by SuperDragonXD — CHANGES_REQUESTED** [review]

File: build.gradle.kts (hunk 1 above)
Comment (ZhmieXD) [reply build.gradle.kts:4]: I know I should've just ignored these PRs until it got updated, oh well ^^;

Anyways, yeah applying. where do I locate the file?

**Comment by Chefski** [comment]
Cheers

**Comment by ZhmieXD** [comment]
> Cheers

Thanks once again! 😄

**Pull request merged**